citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -27,13 +28,33 @@ $(function() {
 	$("#uriLoadingFoundEditor").popover();
 	$("#uriLoadingFailureEditor").popover();
 	
-	$("#submitForm").click(function() {
-		constructPersonArray("author");
-		constructPersonArray("editor");
+	$("#uriLoadingSpinnerCreator").hide();
+	$("#uriLoadingFailureCreator").hide();
+	$("#uriLoadingFoundCreator").hide();
+	
+	$("#uriLoadingFoundCreator").popover();
+	$("#uriLoadingFailureCreator").popover();
+	
+	$("#submitForm").click(function(e) {
+		constructPersonArray("author", "author", 0);
+		constructPersonArray("editor", "editor", 0);
+		var creatorSubmitCount = 0;
+		$(".creator").parent().closest("tr").each(function(idx, elem){
+			if($(elem).css("display")!="none") {
+				var ele = $(elem).children().first();
```

Comment:
instead of this, I would give the element you want a class and get that via the class. Otherwise if we for some reason add a column or something, this will break.